lit-base/model_loader.py

```diff
@@ -0,0 +1,50 @@
+import yaml
+
+import mxnet as mx
+
+import tempfile
+from urllib import urlretrieve
+
+
+# load model
+def load_model(s_fname, p_fname):
+    """
+    Load model checkpoint from file.
+    :return: (symbol, arg_params, aux_params)
+    symbol : Symbol
+        The network configuration.
+    arg_params : dict of str to NDArray
+        Model parameter, dict of name to NDArray of net's weights.
+    aux_params : dict of str to NDArray
+        Model parameter, dict of name to NDArray of net's auxiliary states.
+    """
+    symbol = mx.symbol.load(s_fname)
+    save_dict = mx.nd.load(p_fname)
+    arg_params = {}
+    aux_params = {}
+    for key, value in save_dict.items():
+        tp, name = key.split(':', 1)
+        if tp == 'arg':
+            arg_params[name] = value
+        if tp == 'aux':
+            aux_params[name] = value
+    return symbol, arg_params, aux_params
+
+model_url = yaml.load(open("model_url.yaml", "r"))
+url_params = model_url["url_params"]
+url_symbol = model_url["url_symbol"]
+
+# download params
```

Comment:
can you make a single method `download_file(url)` that returns the file name and just call for both symbol and params

Reply:
Modified accordingly in lit-mar